sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/ForeachSinkSuite.scala

```diff
@@ -141,7 +141,7 @@ class ForeachSinkSuite extends StreamTest with SharedSQLContext with BeforeAndAf
         query.processAllAvailable()
       }
       assert(e.getCause.isInstanceOf[SparkException])
-      assert(e.getCause.getCause.getMessage === "error")
+      assert(e.getCause.getCause.getCause.getMessage === "ForeachSinkSuite error")
```

Comment:
why 3 levels? Can you paste the levels?

Reply:
[info]   org.apache.spark.sql.streaming.StreamingQueryException: Query [id = c80c8860-d4f5-47c6-9a2b-33b5172e1735, runId = 81acd408-9028-41ee-9349-866ae2d67615] terminated with exception: Writing job aborted.
[info]   at org.apache.spark.sql.execution.streaming.StreamExecution.org$apache$spark$sql$execution$streaming$StreamExecution$$runStream(StreamExecution.scala:295)

[info]   Cause: org.apache.spark.SparkException: Writing job aborted.
[info]   at org.apache.spark.sql.execution.datasources.v2.WriteToDataSourceV2Exec.doExecute(WriteToDataSourceV2.scala:117)

[info]   Cause: org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 1.0 failed 1 times, most recent failure: Lost task 0.0 in stage 1.0 (TID 1, localhost, executor driver): java.lang.RuntimeException: ForeachSinkSuite error
[info] 	at org.apache.spark.sql.execution.streaming.sources.ForeachWriterSuite$$anonfun$5$$anonfun$apply$mcV$sp$4$$anon$1.process(ForeachWriterSuite.scala:135)

[info]   Cause: java.lang.RuntimeException: ForeachSinkSuite error
[info]   at org.apache.spark.sql.execution.streaming.sources.ForeachWriterSuite$$anonfun$5$$anonfun$apply$mcV$sp$4$$anon$1.process(ForeachWriterSuite.scala:135)